src/PIL/ImageCms.py

```diff
@@ -254,20 +254,15 @@ def get_display_profile(handle=None):
     :returns: None if the profile is not known.
     """
 
-    if sys.platform == "win32":
-        from PIL import ImageWin
+    if sys.platform != "win32":
+        return
```

Comment:
```suggestion
        return None
```
Let's explicitly return None, as this is a function that returns something.